play-services-core/src/main/res/xml/preferences_start.xml

```diff
@@ -65,9 +65,15 @@
             android:title="@string/service_name_location"/>
     </PreferenceCategory>
     <PreferenceCategory android:layout="@layout/preference_category_no_label" android:key="prefcat_footer">
+        <SwitchPreferenceCompat
```

Comment:
I don't think this setting is so important that it deserves a prime location at the top level settings dialog. I suggest to create a new section for these kind of things, although I don't know how to name it or what else would belong there.

Reply:
@mar-v-in
Some ideas:
1) Other settings
2) Minor settings
3) Trivial settings

Reply:
What about "Miscellaneous"?